topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -170,140 +157,297 @@ To estimate sequence quality along all sequences, we now use [FastQC](https://ww
+>   
+> The plot shows the deviation from the average quality for each tile. The colours are on a cold to hot scale, with cold colours being positions where the quality was at, or above, the average for that base in the run, and hotter colours indicate that a tile had worse qualities than other tiles for that base. In the example above you can see that certain tiles show consistently poor quality. A good plot should be blue all over.
 >
->     It's not unusual to see a very low proportion of Ns appearing in a sequence, especially near the end of a sequence. However, if this proportion rises above a few percent it suggests that the analysis pipeline was unable to interpret the data well enough to make valid base calls.
+> #### Per base N content
 >
-> - **Sequence length distribution**
+> ![Per base N content](../../images/quality-control/per_base_n_content.png "Per base N content")
 >
->     ![Sequence length distribution](../../images/quality-control/sequence_length_distribution.png "Sequence > length distribution")
+> If a sequencer is unable to make a base call with sufficient confidence then it will normally substitute an N rather than a conventional base call. This plot displays the percentage of base calls at each position or bin for which an N was called.
 >
->     Some high throughput sequencers generate sequence fragments of uniform length, but others can contain reads of wildly varying lengths. Even within uniform length libraries some pipelines will trim sequences to remove poor quality base calls from the end.
+> It's not unusual to see a very low proportion of Ns appearing in a sequence, especially near the end of a sequence. But this curve should never rises noticeably above zero. If it does this indicates a problem occurred during the sequencing run. In the example below, an error caused the instrument to be unable to call a base for approximately 20% of the reads at position 29:
 >
->     This graph shows the distribution of fragment sizes in the file which was analysed. In many cases this will produce a simple graph showing a peak only at one size, but for variable length FASTQ files this will show the relative amounts of each different size of sequence fragment.
+> ![Per base N content](../../images/quality-control/per_base_n_content_error.png)
+> 
+> #### Sequence length distribution
 >
-> - **Sequence Duplication Levels**
+> ![Sequence length distribution](../../images/quality-control/sequence_length_distribution.png "Sequence length distribution")
 >
->     ![Sequence Duplication Levels](../../images/quality-control/sequence_duplication_levels.png "Sequence Duplication Levels")
+> Some high throughput sequencers generate sequence fragments of uniform length, but others can contain reads of wildly varying lengths. Even within uniform length libraries some pipelines will trim sequences to remove poor quality base calls from the end.
```

Comment:
For instance the Illumina base caller (bcl2fastq) does adapter trimming (by default) in a way that the first n bases are trimmed if they matche the first n bases of the adapter up to 90% (by default). The problem is that n can also be 1. 

In the plot you see then that ~1/4 of the sequences are shorter by 1, ~1/16 of the data shorter by 2, ....

So we disabled this trimming now and use cutadapt.